src/main/java/Duke.java

```diff
@@ -1,10 +1,167 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+    private static final String GREETING = "Hello! I'm Duke\nWhat can I do for you?";
+    private static final String GOODBYE = "Bye. Hope to see you again soon!";
+    private static final ArrayList<Task> TASKS = new ArrayList<>();
+
+    /**
+     * @param txt text to indent.
+     * @return indented string.
+     */
+    private static String autoIndent(String txt) {
+        return "    " + txt.replace("\n", "\n    ");
+    }
+
+    /**
+     * prints border line.
+     */
+    private static void borderLine() {
+        System.out.print("    ____________________________________________________________\n");
+    }
+
+    /**
+     * Prints output in a nice format. (adds borders and indentation).
+     *
+     * @param inp input string.
+     */
+    private static void prettifyOut(String inp) {
+        borderLine();
+        System.out.println(autoIndent(inp));
+        borderLine();
+    }
+
+    /**
+     * Prints out that task was added.
+     *
+     * @param task task that was added.
+     */
+    private static void printTaskAdded(Task task) {
+        prettifyOut("Got it. I've added this task:\n  " + task + "\nNow you have " + TASKS.size() +
+                " tasks in the list.");
+    }
+
+    /**
+     * Executes the input command.
+     * @param inp input string.
+     * @return False if input is a bye command.
+     *
+     * @throws DukeException if input is invalid.
+     */
+    private static boolean executeInput(String inp) throws DukeException {
```

Comment:
Since this method is a boolean wouldn't it be better name it in a boolean sounding way?